### PR TITLE
Modified the weather package to use IBM Weather Insights. Removed po…

### PR DESCRIPTION
**Pull request by nwspeete-ibm**
…llen.js as IBM Weather Insights does not yet have a pollen API.

### CODE REVIEW

File: catalog/installWeather.sh [hunk 1]
```diff
@@ -12,24 +12,18 @@ source "$CATALOG_HOME/util.sh"
...
     -a parameters '[ {"name":"apiKey", "required":false} ]'
 
 waitForAll
 
 install "$CATALOG_HOME/weather/forecast.js" \
     weather/forecast \
-    -a description 'Weather.com 10-day forecast' \
-    -a parameters '[ {"name":"latitude", "required":true}, {"name":"longitude", "required":true}, {"name":"apiKey", "required":true, "type":"password", "bindTime":true} ]' \
+    -a description 'IBM Weather Insights 10-day forecast' \
+    -a parameters '[ {"name":"latitude", "required":true}, {"name":"longitude", "required":true},{"name":"language", "required":true},{"name":"units", "required":true}, {"name":"apiKey", "required":true, "type":"password", "bindTime":true} ]' \
```
Comment (rabbah) [inline catalog/installWeather.sh]: some of these annotations carry the `required` attribute but the `sampleInput` does not specify them (e.g., `language` and `units`).

File: catalog/weather/forecast.js [hunk 2]
```diff
@@ -12,28 +12,19 @@ var request = require('request');
...
-    var url = 'https://api.weather.com/v1';
-    if (zip)
-        url += '/location/' + zip + ':4:US';
-    else
-        url += '/geocode/' + lat + '/' + lon;
-    url += '/forecast/daily/10day.json'
-         + '?language=' + language
-         + '&apiKey=' + apiKey
-         + '&units=' + units;
+    var url = 'https://' + apiKey + '@twcservice.mybluemix.net:443/api/weather/v2/forecast/daily/10day?units=' + units + '&geocode=' + lat + '%2C' + lon + '&language=' + language;
```
Comment (rabbah) [inline catalog/weather/forecast.js]: `443` not required since protocol is `https`?

**Comment by nwspeete-ibm** [comment]
Took out the 443. I will add a forecast parameter for all API options:
1.GET /api/weather/v2/geocode/forecast/daily/10day
Returns a daily 10-day forecast, for a geolocation, in the provided units of measure

2.GET /api/weather/v2/geocode/forecast/hourly/24hour
Returns an hourly 2-day forecast, for a geolocation, in the provided units of measure

3.GET /api/weather/v2/geocode/observations/current
Returns the current weather conditions, for a geolocation, in the provided units of measure

4.GET /api/weather/v2/geocode/observations/timeseries/24hour
Returns both the current observations and up to 24 hours of past observations, from the current date and time, for a geolocation, in the provided units of measure

**Comment by nwspeete-ibm** [comment]
@rabbah - Forecast time period parameter added and ready for review.

**Comment by csantanapr** [comment]
@rabbah Can you review/merge this please, Anything blocking this PR from being merge?

@nwspeete-ibm Can you rebase the PR, Github is reporting that is out-of-date

**Comment by rabbah** [comment]
In general, most times the person doing the merge can rebase and ff without resolving conflicts. This avoids the added overhead of constantly rebasing. Unless you know there is a conflict, you can skip rebasing. Since we don't use the `merge` button, ever.

**Comment by rabbah** [comment]
Rebased and merged as [0a5a52d](https://github.com/openwhisk/openwhisk/commit/0a5a52d8fb70097ef4f2479d4b8da06093f7b411).

**Pull request closed without merging**